examples/form/src/App.js

```diff
@@ -25,23 +26,31 @@ class App extends Component {
         min: 0.25,
         max: 0.75
       },
-      steps: new Array(21).fill().map((e,i) => i * 0.05)
+      steps: new Array(21).fill().map((e,i) => i * 0.05),
+      textbox: ''
     };
     this.clickButton = this.clickButton.bind(this);
     this.onChangeCheckbox1 = this.onChangeCheckbox1.bind(this);
     this.onChangeCheckbox2 = this.onChangeCheckbox2.bind(this);
+    this.onChangeTextbox = this.onChangeTextbox.bind(this);
     this.onChangeSinglePicker = this.onChangeSinglePicker.bind(this);
     this.onChangeMultiplePicker = this.onChangeMultiplePicker.bind(this);
     this.onMouseLeaveRow = this.onMouseLeaveRow.bind(this);
     this.onMouseEnterRow = this.onMouseEnterRow.bind(this);
     this.setSliderHandleChange = this.setSliderHandleChange.bind(this);
   }
 
-  onChangeCheckbox1() {
+  // The functions that are called when change directlly
+  // the textbox or checkbox recive an event as param
+  onChangeTextbox(event) {
+    this.setState({ [event.target.name]: event.target.value });
```

Comment:
I think this should be `{ textbox: event.target.value } `as no other component is setting the value using the `event.target.name` attribute

Reply:
I thought in that, but if someone add another texbox on the example, is necessary create another function for change the text, tacking the name of the component we can reuse this function.

Another case is with the checkbox, we can write a function like: `changeCheckbox(event) { this.setState({ [event.target.name]: event.target.checked }) }`, this function works when change the checkbox directly, but not when press a button

Reply:
I think its ok to leave it this way, since it is the `examples` subproject and it hints users to manage the state using the target's name.